{"name": "# C5188 Device Dashboard page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <i _ngcontent-epc-c5=\"\" class=\"fa fa-hdd-o\"></i> is not clickable at point (30, 302). Other element would receive the click: <div class=\"swal2-container swal2-center swal2-fade swal2-shown\" style=\"overflow-y: hidden;\">...</div>\n  (Session info: chrome=87.0.4280.66)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5176_Add_a_new_device.py\", line 8, in go_to_dvcs_pg\n    context.app.main_page.go_to_dvcs_pg(dvcs_url)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 1032, in go_to_dvcs_pg\n    wait.until(EC.element_to_be_clickable(DVCS_ICN)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1606886340196, "stop": 1606886346049}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1606886346050, "stop": 1606886348880}, {"name": "And Go to the Devices page https://devcloud.connectedio.com/devices", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <i _ngcontent-epc-c5=\"\" class=\"fa fa-hdd-o\"></i> is not clickable at point (30, 302). Other element would receive the click: <div class=\"swal2-container swal2-center swal2-fade swal2-shown\" style=\"overflow-y: hidden;\">...</div>\n  (Session info: chrome=87.0.4280.66)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5176_Add_a_new_device.py\", line 8, in go_to_dvcs_pg\n    context.app.main_page.go_to_dvcs_pg(dvcs_url)\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 1032, in go_to_dvcs_pg\n    wait.until(EC.element_to_be_clickable(DVCS_ICN)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1606886348880, "stop": 1606886349997}, {"name": "Then Click Add new device button on the right up corner", "status": "skipped", "start": 1606886352072, "stop": 1606886352072}, {"name": "Then Fill required Device IMEI field 356961071557824", "status": "skipped", "start": 1606886352072, "stop": 1606886352072}, {"name": "And Fill Device Name field if any TVM Test123 jjnkjn", "status": "skipped", "start": 1606886352073, "stop": 1606886352073}, {"name": "Then Fill required Serial Number field 1806208A0279", "status": "skipped", "start": 1606886352073, "stop": 1606886352073}, {"name": "And Click Save and refresh the page", "status": "skipped", "start": 1606886352073, "stop": 1606886352073}, {"name": "Then Success is here - device is on the page", "status": "skipped", "start": 1606886352073, "stop": 1606886352073}, {"name": "Then Choose any device with IMEI number which is highlighted in blue active IMEI number and click on an active IMEI number", "status": "skipped", "start": 1606886352073, "stop": 1606886352073}, {"name": "Then Verify the Device Management Portal has opened", "status": "skipped", "start": 1606886352073, "stop": 1606886352073}, {"name": "And Delete device", "status": "skipped", "start": 1606886352073, "stop": 1606886352073}], "start": 1606886335031, "stop": 1606886352073, "uuid": "e74e6950-b8d1-410f-adde-0603570466f0", "historyId": "06f5543c6cdade6c93c93ff17d488c0a", "fullName": "features/tests/Device Dashboard/C5188_Device_Dashboard_page.feature:# C5188 Device Dashboard page", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5188 Device Dashboard page"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}